doc/releases.md

```diff
@@ -1,5 +1,115 @@
 # Releases
 
+## Version 1.19
+
+### Version 1.19.0
+
+**June 13, 2024**
+
+This release includes an exciting new feature popups, this will make it easier to dive deeper and interact with your data.
```

Comment:
This sentence also doesn't read quite right "includes an exciting new feature popups" isn't grammatically correct.

```suggestion
This release introduces a number of major new features and enhancements including the ability to add popups, allowing you to easily drill down into subsets of your data. Additionally it finally allows HoloViews to retain and use pandas (multi-)indexes, providing major benefits for working with wide (as opposed to tidy) data.  
```